gradle/libs.versions.toml

```diff
@@ -47,5 +47,5 @@ truth = { module = "com.google.truth:truth", version = "1.1.3" }
 compileTesting = { module = "com.google.testing.compile:compile-testing", version = "0.21.0" }
 jimfs = { module = "com.google.jimfs:jimfs", version = "1.2" }
 ecj = { module = "org.eclipse.jdt.core.compiler:ecj", version = "4.6.1" }
-kotlinCompileTesting = { module = "com.github.tschuchortdev:kotlin-compile-testing", version.ref = "kct" }
-kotlinCompileTesting-ksp = { module = "com.github.tschuchortdev:kotlin-compile-testing-ksp", version.ref = "kct" }
+kotlinCompileTesting = { module = "dev.zacsweers.kctfork:core", version.ref = "kct" }
```

Comment:
Curious, what are these forks for?

Reply:
Original project is seemingly abandoned and wasn't getting updated, so I'm maintaining a fork now.